cli/tests/default/unit/snapshots/test_rule_match/test_rule_match_to_app_finding_historical_info/results.json

```diff
@@ -0,0 +1,22 @@
+{
+  "check_id": "rule.id",
+  "column": 0,
+  "commit_date": "1970-01-01T00:00:00",
+  "end_column": 0,
+  "end_line": 0,
+  "hashes": {
+    "code_hash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+    "end_line_hash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+    "pattern_hash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+    "start_line_hash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+  },
+  "index": 0,
+  "is_blocking": true,
+  "line": 0,
+  "match_based_id": "ac39ef6548f7e4407744659d806fdcd7ed74bff05adf3663a15a6f58a0207a628bf9bc7dd83c598afda27a848c236df8d179557e064e82d8bd776020635992ad_0",
+  "message": "message",
+  "metadata": {},
```

Comment:
I would expect to see historical info somewhere in this snapshot 🤔

Reply:
Yeah, I thought I fixed it. Let me double check I pushed the last commit given this + the prints.